main/grub/grub.trigger

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+if [ -e /boot/vmlinuz-vanilla ]; then
```

Comment:
I wonder if we should check if `/boot/grub/grub.cfg` exists too. You can install grub without having any `/boot/grub/grub.cfg`